satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -1,5 +1,18 @@
 // dbx.v1 golang satellitedb.dbx .
 
+//--- Value Attribution ---//
+model value_attribution (
+	key bucket_id
+
+	field bucket_id		blob
+	field user_id		blob
+	field partner_id	blob
+	field at_rest_data	int64
+	field egress_data	int64
+	field ingress_data  	int64
```

Comment:
these values should probably not be denormalized here. instead, when collections are normally due, we should just join with this bucket when doing the usage report to see who gets a cut. so i propose dropping the three int64 fields

Reply:
I don't understand the why

Reply:
keeping these fields up to date increases the database load for no reason - we only pay partners when we're doing invoicing anyway, so let's delay the running calculation of how much usage there is for the invoicing and usage processes.